### DISCUSSION TITLE
what System of Measurement does openvslam output follow

### DISCUSSION THREAD

**Discussion by spdjohn123** [stella-cv/stella_vslam#170] [Q&A] [answered]
Hello,

I am a student from Germany. I was just interested on what kind of Measurement system does openvslam output follow?
Can we represent it in any of the two available measurement system to understand the world coordinate better?
If so then how we can represent or convert into imperial or metric system? It will be helpful for my final year project.

**Comment by ryleymcc** [comment, marked as answer]
I would imagine it would be unit-less system until you add a reference. A stereo baseline should technically be good enough for a reference but could also be aided by IMU measurement which is not implemented.
